play-2.4/swagger-play2/app/play/modules/swagger/PlayReader.java

```diff
@@ -579,7 +601,7 @@ private Type getParamType(Class<?> cls, Method method, String simpleTypeName) {
                 if (schema != null) ((QueryParameter)parameter).setProperty(schema);
             }
             parameter.setName(p.name());
-            List<Annotation> annotations = getParamAnnotations(cls, method, p.typeName());
+            List<Annotation> annotations = getParamAnnotations(cls, method, p.typeName(), fieldPosition++);
```

Comment:
I think it would be slightly clearer to increment fieldPosition in a separate operation, since the incrementing has nothing really do to with this line